### PR TITLE
feat: maskbox theme

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-722

## Type of change

- [ ] Documentation
- [x] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 1]
```diff
@@ -90,6 +148,8 @@ export function PreviewCard(props: PreviewCardProps) {
         retryMaskBoxPurchasedTokens,
     } = useContainer(Context)
 
+    const [currentTab, onChange, tabs] = useTabs('Articles', 'Details')
```
Comment (guanbinrui) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx:151]: i18n

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 2]
```diff
@@ -171,78 +231,150 @@ export function PreviewCard(props: PreviewCardProps) {
...
+                                variant="roundedDark"
+                                fullWidth
+                                disabled={boxState !== BoxState.READY || isOpening}
+                                onClick={() => setOpenDrawDialog(true)}>
+                                {(() => {
+                                    return boxState === BoxState.READY && paymentTokenAddress ? (
+                                        <>
+                                            {boxStateMessage} (
+                                            {formatBalance(paymentTokenPrice, paymentTokenDetailed?.decimals ?? 0)}{' '}
+                                            {paymentTokenDetailed?.symbol}/Time)
```
Comment (guanbinrui) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]: i18n

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx (hunk 1 above)
Comment (lelenei) [reply packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx:151]: unnecessary

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/DetailsTab.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ const useStyles = makeStyles()((theme) => ({
     main: {
         height: 360,
         overflow: 'auto',
-        marginBottom: theme.spacing(2.75),
+        padding: 16,
```
Comment (UncleBill) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/DetailsTab.tsx]: use `theme.spacing` to ensure consistency of spacing.
```suggestion
        padding: theme.spacing(2),
```

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 4]
```diff
@@ -12,32 +12,13 @@ import { DrawResultDialog } from './DrawResultDialog'
 import { useTransactionCallback, TargetChainIdContext } from '@masknet/plugin-infra/web3-evm'
 import { ChainBoundary } from '../../../../web3/UI/ChainBoundary'
 import { formatBalance, NetworkPluginID } from '@masknet/web3-shared-base'
-import type { AbstractTabProps } from '../../../../components/shared/AbstractTab'
-import AbstractTab from '../../../../components/shared/AbstractTab'
+import { TabContext } from '@mui/lab'
+import { formatAddress } from '@masknet/web3-shared-flow'
```
Comment (UncleBill) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]: At least use `formatAddress` for EVM.
```suggestion
import { formatAddress } from '@masknet/web3-shared-evm'
```

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 5]
```diff
@@ -49,6 +29,82 @@ const useTabsStyles = makeStyles()((theme) => ({
...
+    tab: {
+        whiteSpace: 'nowrap',
+        background: 'transparent',
+        color: theme.palette.maskColor.second,
+        '&:hover': {
+            background: 'transparent',
+        },
+    },
+    tabActive: {
+        background: '#fff',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]: Avoid using hex string color.

```suggestion
        backgroundColor: '#fff',
```

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 6]
```diff
@@ -49,6 +29,82 @@ const useTabsStyles = makeStyles()((theme) => ({
...
+        height: 50,
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        position: 'relative',
+    },
+    name: {
+        color: theme.palette.maskColor.main,
+        fontSize: 18,
+        fontWeight: 'bold',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]: Use an absolute value.

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 7]
```diff
@@ -49,6 +29,82 @@ const useTabsStyles = makeStyles()((theme) => ({
...
+        margin: '0 12px',
+        flex: 1,
+        backgroundColor: theme.palette.maskColor.white,
+        overflow: 'auto',
+        borderRadius: '0 0 12px 12px',
+        scrollbarWidth: 'none',
+        '&::-webkit-scrollbar': {
+            display: 'none',
+        },
+        background: '#fff !important',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]: Duplicated

**Review by guanbinrui — DISMISSED** [review]

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 8]
```diff
@@ -173,78 +231,149 @@ export function PreviewCard(props: PreviewCardProps) {
...
+                                </Typography>
+                                <Typography
+                                    sx={{ paddingRight: 1, paddingLeft: 1 }}
+                                    color={theme.palette.maskColor.second}
+                                    fontSize={14}
+                                    fontWeight={400}>
+                                    {t.limit()}
+                                </Typography>
+                                <Typography color="textPrimary" fontSize={14} fontWeight="bold">
+                                    {1}
```
Comment (UncleBill) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]:
```suggestion
                                    1
```

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 9]
```diff
@@ -173,78 +231,149 @@ export function PreviewCard(props: PreviewCardProps) {
...
+                                <Typography
+                                    sx={{ paddingRight: 1 }}
+                                    color={theme.palette.maskColor.second}
+                                    fontSize={14}
+                                    fontWeight={400}>
+                                    {t.by()}
+                                </Typography>
+                                <Typography color="textPrimary" fontSize={14} fontWeight="bold">
+                                    {Others?.formatAddress(boxInfo.creator, 4)}
+                                </Typography>
```
Comment (UncleBill) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]:
```suggestion
                                <Typography color="textPrimary" fontSize={14} fontWeight="bold" title={boxInfo.creator}>
                                    {Others?.formatAddress(boxInfo.creator, 4)}
                                </Typography>
```

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 10]
```diff
@@ -173,78 +231,149 @@ export function PreviewCard(props: PreviewCardProps) {
...
+                    </Box>
+                    {boxState === BoxState.READY ? (
+                        <Typography className={tabClasses.active}>{t.active()}</Typography>
+                    ) : null}
+                </Box>
+                <Box className={tabClasses.body}>
+                    <MaskTabList variant="base" aria-label="maskbox" onChange={onChange}>
+                        {Tabs.map((x, i) => (
+                            <Tab
+                                key={i}
```
Comment (UncleBill) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx]:
```suggestion
                                key={x.value}
```

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx (hunk 1 above)
Comment (Jack-Works) [reply packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx:151]: the input for this hook is not for display, but more like an enum. therefore it does not need i18n

File: packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx [hunk 11]
```diff
@@ -173,78 +231,149 @@ export function PreviewCard(props: PreviewCardProps) {
...
+                                </Typography>
+                                <Typography
+                                    sx={{ paddingRight: 1, paddingLeft: 1 }}
+                                    color={theme.palette.maskColor.second}
+                                    fontSize={14}
+                                    fontWeight={400}>
+                                    {t.limit()}
+                                </Typography>
+                                <Typography color="textPrimary" fontSize={14} fontWeight="bold">
+                                    1
```
Comment (Jack-Works) [inline packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx:284]: 🤔?

**Pull request merged**